scripts/covid19indiaORG/Config.py

```diff
@@ -0,0 +1,177 @@
+state_apis = [
+ {
+   "Abbreviation": "WB",
+   "State": "West Bengal",
```

Comment:
Do we need the "State" field?

Is the API always the same? `https://api.covid19india.org/v4/min/timeseries-<Abbreviation>.min.json` ? I am wondering if we can just use a list of Abbreviation.

Reply:
The reason for the State field is because the DISTRICTS map is keyed by State->District.
The API the District Names, I need the name of the State.

Although, another workaround is to key the DISTRICTS map by abbreviation->district.

Reply:
I see. So Abbreviation is not unique within India? I was hoping the values correspond to the isoCodes https://screenshot.googleplex.com/6dSiQeFsXniXqu3

Reply:
This might be a stretch, but IF these are isoCodes, it would be nice to give isoCodes for states, since there is more certainty. For districts, if there are no codes, we can continue with your name to wikidata approach.

Reply:
Upon looking, the abbreviation is actually the iso code minus the IN- in front.
The iso code is IN-WB but they use WB.

I have changed the INDIA hashmaps to be referenced by iso code instead of the name.

So now, they are hashed by: stateIsoCode -> district_name -> wikidataId.

Reply:
<img width="683" alt="Screen Shot 2020-12-02 at 12 44 08 PM" src="https://user-images.githubusercontent.com/11444894/100910498-13671f80-349c-11eb-8021-2f8416f7a1c5.png">

Reply:
Should I add the IN- infront? Their APIs don't use the IN-.

Reply:
no that's fine, I was thinking we could resolve with isoCode, but if you manually mapped iso<>wikidataId, then it should be good.